en/advanced-statefulset.md

```diff
@@ -36,7 +36,7 @@ The [advanced StatefulSet controller](https://github.com/pingcap/advanced-statef
 
 2. Enable the `AdvancedStatefulSet` feature in `values.yaml` of the TiDB Operator chart:
 
-    {{< copyable "shell-regular" >}}
+    {{< copyable "yaml" >}}
```

Comment:
For `yaml`, please use `{{< copyable "" >}}`. Thanks!

Reply:
ok